Amethyst/Managers/FocusTransitionCoordinator.swift

```diff
@@ -159,6 +161,90 @@ class FocusTransitionCoordinator<Target: FocusTransitionTarget> {
         // Otherwise focus the topmost window
         target.executeTransition(.focusWindow(topWindow))
     }
+
+    func moveFocusScreenCounterClockwise() {
```

Comment:
This seems to duplicate behavior above in `focusScreen`. Maybe just get the index and call that method instead?